app/classes/Transvision/API.php

```diff
@@ -299,6 +299,11 @@ private function isValidServiceCall($service)
                     No user-defined variables = nothing to check
                 */
                 break;
+            case 'transliterate':
```

Comment:
this case should be before the 'version' case, the services in the switch are sorted alphabetically.
Also, there should be a comment indicating an API call example.